### PR TITLE
Trail build logs if follow enabled for envs edit

### PR DESCRIPTION
**Pull request by fuskovic**
### What this does

Trails build logs when `--follow` is enabled for `coder envs edit`

### CODE REVIEW

File: internal/cmd/envs.go [hunk 1]
```diff
@@ -298,6 +298,12 @@ coder envs edit back-end-env --disk 20`,
 				clog.BlankLine,
 				clog.Tipf(`run "coder envs watch-build %q" to trail the build logs`, envName),
```
Comment (cmoog) [inline internal/cmd/envs.go:299]: Shouldn't we only print this if `!follow`?

File: docs/coder.md [hunk 2]
```diff
@@ -2,10 +2,6 @@
 
 coder provides a CLI for working with an existing Coder Enterprise installation
 
-### Synopsis
-
-coder provides a CLI for working with an existing Coder Enterprise installation
-
```
Comment (coadler) [inline docs/coder.md:8]: Can we have a separate commit explaining why these were removed? Odd for it to be bundled into the other commit.

**Review by coadler — APPROVED** [review]

File: docs/coder.md (hunk 2 above)
Comment (cmoog) [reply docs/coder.md:8]: Looks like it's because we updated `cobra`. CI enforces that the docs are properly generated and always up-to-date.

Comment (coadler) [reply docs/coder.md:8]: Ah, didn't realize this was generated. sg

**Review by cmoog — APPROVED** [review]

**Pull request merged**